Sources/Atomic.swift

```diff
@@ -125,27 +125,37 @@ internal struct UnsafeAtomicState<State: RawRepresentable>: AtomicStateProtocol
 #endif
 }
 
-final class PosixThreadMutex: NSLocking {
-	private var mutex = pthread_mutex_t()
+internal struct PosixThreadMutex {
+	private let mutex: UnsafeMutablePointer<pthread_mutex_t>
```

Comment:
Can you add a comment that explains this?

Any time we choose the non-obvious route to get a speed boost, we should document it.